.travis.yml

```diff
@@ -26,8 +26,19 @@ cache:
 before_cache:
   - rm -rf /home/travis/.cargo/registry
 
+before_install:
+  - if [[ $TRAVIS_OS_NAME == "linux" ]]; then export PATH=$HOME/protobuf/bin:$PATH; fi
+  - if [[ $TRAVIS_OS_NAME == "linux" ]]; then export C_INCLUDE_PATH=$C_INCLUDE_PATH:$HOME/protobuf/include; fi
+  - if [[ $TRAVIS_OS_NAME == "linux" ]]; then export CPLUS_INCLUDE_PATH=$CPLUS_INCLUDE_PATH:$HOME/protobuf/include; fi
 
 install:
+  - >
+    if [[ $TRAVIS_OS_NAME == "linux" ]]; then
+        mkdir -p $HOME/protobuf && pushd $HOME/protobuf &&
```

Comment:
How can test pass in the past?

Reply:
This PR adds `rustproto.proto` in `proto/include`. The compiler needs to compile it with `protoc`.